JitsiConference.js

```diff
@@ -1377,6 +1382,37 @@ JitsiConference.prototype.onIncomingCall
 };
 
 /**
+ * Sets the BridgeChannel.
+ */
+JitsiConference.prototype._setBridgeChannel = function() {
+    const jingleSession = this.jvbJingleSession;
```

Comment:
you should check if this is not null / undefined, since it might not exist in case of a p2p connection.